.jshintrc

```diff
@@ -5,9 +5,9 @@
 "eqnull" : true,
 "indent": 2,
 "undef": true,
-"unused": true,
-"quotmark": "single",
-"maxlen": 90,
+"unused": false,
+"quotmark": "true",
+"maxlen": 110,
```

Comment:
I had to extend the line length to work around the JSDoc issue. For example, 
```js
/**
 * Build SQL expression
 * @param {String} columnName Escaped column name
 * @param {String} operator SQL operator
 * @param {*} columnValue Column value
 * @param {*} propertyValue Property value
 * @returns {ParameterizedSQL} The SQL expression
 */
SqlConnector.prototype.buildExpression = function(columnName, operator, columnValue, propertyValue) {
```

If I split the line from `= function(...)`, strong-docs will not show the JSDoc.